scene/gui/spin_box.cpp

```diff
@@ -280,8 +280,8 @@ void SpinBox::_notification(int p_what) {
 		} break;
 
 		case NOTIFICATION_THEME_CHANGED: {
-			call_deferred(SNAME("update_minimum_size"));
-			get_line_edit()->call_deferred(SNAME("update_minimum_size"));
+			callable_mp((Control *)this, &Control::update_minimum_size).call_deferred();
+			callable_mp((Control *)get_line_edit(), &Control::update_minimum_size).call_deferred();
```

Comment:
This is already called automatically when `NOTIFICATION_THEME_CHANGED` is received, the only difference is that here it's deferred for both nodes. I wonder if it could be removed.